runtime/lua/vscode/files.lua

```diff
@@ -0,0 +1,23 @@
+local api = vim.api
+local vscode = require("vscode.api")
+
+local M = {}
+
+function collect_file_info(event)
+  return {
+    buf = event.buf,
+    path = event.file,
+  }
+end
+
+function M.setup()
+  api.nvim_create_autocmd({ "BufWriteCmd" }, {
+    pattern = "*",
+    callback = function(event)
+      local info = collect_file_info(event)
+      vscode.action("save_buf", { args = { info.buf, info.path } })
```

Comment:
Maybe it's better to send an object.

Reply:
Yeah agreed. I just did this because it was consistent with other examples I saw.